client/src/index.ts

```diff
@@ -84,6 +87,7 @@ function requestUpdate() {
     /* Main update rate = 250ms is minimum time, equal to server update time. */
     getUnits((data: UnitsData) => {
         getUnitsManager()?.update(data);
+        unitDataTable.refresh( data.units );
```

Comment:
For consistency, please add a "getDataTable()" method. This allows to easily disable the updates if needed.

Reply:
I'll take a look.